style.css

```diff
@@ -72,6 +72,9 @@ body {
   justify-content: space-between;
   position: absolute;
   left: 0;
+  height: 100px;
+
+  /* background-color: #396df2; */
```

Comment:
- [ ] Beautiful! All you just have to do now is to remove this new comment that you just made. Kindly remember that keeping old pieces of code in your workflow as comments is against best practices, as it can potentially create bugs or make code difficult to read :wink: